test/integration/visitor_can_add_item_to_cart_and_view_cart_test.rb

```diff
@@ -26,7 +36,9 @@ class VisitorCanAddItemToCartAndViewCartTest < ActionDispatch::IntegrationTest
     click_link 'View Item'
     click_link 'Add To Cart'
 
-    assert page.has_content?('Cart: 1')
+    within('.cart-count') do
+      assert page.has_content?('1')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.